flask/datahandle.py

```diff
@@ -2,7 +2,10 @@
 import requests
 import urllib.request
 
-#remove spaces in url string
+
+###Helper functions
```

Comment:
If you want to improve these comments you could look into [Sphinx](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html). It generates nice documentation on something like readthedocs.

Autodocstring (VSCode) makes this auto complete, is pretty nice. 
https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring

Reply:
Oh wow, I'll look into that next week.

Reply:
I'll create this as a new issue